exams/signals_test.py

```diff
@@ -73,10 +78,16 @@ def test_update_exam_profile_called(self):
 
         assert profile_exam.status == ExamProfile.PROFILE_PENDING
 
+    @override_settings(FINAL_GRADE_ALGORITHM='v1')
     def test_update_exam_authorization_final_grade(self):
         """
         Verify that update_exam_authorization_final_grade is called when a FinalGrade saves
         """
+        create_order(self.profile.user, self.course_run)
+        with mute_signals(post_save):
+            # muted because enrollment also trigger signal for profile creation. right now we are just
+            # looking final grades
+            CachedEnrollmentFactory.create(user=self.profile.user, course_run=self.course_run)
         final_grade = get_final_grade(self.profile.user, self.course_run)
```

Comment:
I'd add that here is probably another case where you want to assert that there is no `ExamProfile` or `ExamAuthorization` before creating the `FinalGrade`.